ddl/lightning/mem_root.go

```diff
@@ -30,22 +31,48 @@ type MemRoot interface {
 	MaxMemoryQuota() int64
 	CurrentUsage() int64
 	CurrentUsageWithTag(tag string) int64
+	RefreshConsumption()
+}
+
+const (
+	_kb = 1024
```

Comment:
https://github.com/pingcap/tidb/blob/11dbd0b881671db0110e90dec3e0ebeceaad5fc1/executor/inspection_result.go#L428-L432

It is the same as here. so we can put them into ```util```.